lib/elixir/lib/exception.ex

```diff
@@ -1631,3 +1631,90 @@ defmodule ErlangError do
   defp nth(3), do: "3rd"
   defp nth(n), do: "#{n}th"
 end
+
+defmodule Inspect.Error do
+  @moduledoc """
+  Raised when a struct cannot be inspected.
+  """
+  # QUESTION: I am not sure if is it considered backward incompatible
+  # to enforce keys that did not exist before
```

Comment:
It is not a breaking change because Inspect.Error is never returned, it is only always printed. So you can remove the `:message` field.